tests/QueuedJobsTest/TestQJService.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace SilverStripe\QueuedJobs\Tests;
+
+use SilverStripe\QueuedJobs\Services\QueuedJobService;
+
+// stub class to be able to call init from an external context
+class TestQJService extends QueuedJobService
```

Comment:
If I add "TestOnly" to these objects, the tests fail as SapphireTest ignores buildings the database for these objects.

https://github.com/silverstripe/silverstripe-framework/blob/master/src/Dev/SapphireTest.php#L1173